### PR TITLE
New package: sys-firmware/midisport-firmware

### PR DESCRIPTION
**Pull request by lilrc**
Modified import from proaudio overlay. Fixes gentoo bug 253988. I have added myself as proxy-maintainer.

@proxy-maint

### CODE REVIEW

File: sys-firmware/midisport-firmware/midisport-firmware-1.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+RDEPEND="sys-apps/fxload
+	virtual/udev"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+CONFIG_CHECK="~SND_USB_AUDIO"
+
+src_prepare() {
+	eapply "${FILESDIR}"/${P}-configure.patch
```
Comment (monsieurp) [inline sys-firmware/midisport-firmware/midisport-firmware-1.2.ebuild]: No. Please put those two patches in an array. By convention we call this array PATCHES. Then, within src_prepare(), you must call the epatch function and stringify the array like so: https://github.com/gentoo/gentoo/blob/335e115d3a96663ddced605dbb91b9eace608d4a/dev-java/colt/colt-1.2.0-r3.ebuild#L36.

**Comment by lilrc** [comment]
Alright. Done. I found that `default` did the job (I did not just try, I read the EAPI reference too) and it was more concise than:
```
eapply "${PATCHES[@]}"
eapply_user
```

**Comment by idella** [comment]
If this is a new package and you have added a new license, the license will need adding to the folder licenses of portage.

**Comment by idella** [comment]
Also there is a question of whether it belongs to a license group and needs adding the the corresponding files  in profiles

**Comment by lilrc** [comment]
I added `licenses/Midisport` and the licesense does not fit in any of the categories. It is not FSF-APPROVED, nor OSI-APPROVED or MISC-FREE due to
```
The firmware files (*.ihx) are copyrighted by Midiman, and can be used
and redistributed only as part of this package.
```
Of the same reason it is not BINARY-REDISTRIBUTABLE. Also it is not EULA since the above license statement does not take away anyone's rights.

**Comment by lilrc** [comment]
I believe the QA failure is not my fault. Is there anything preventing this from being merged?

**Comment by lilrc** [comment]
I guess a build log is appropriate:
https://karll.portknox.net/index.php/s/PY78fLUvcDHkXuX

**Comment by monsieurp** [comment]
Merged in 0061153615b27e6e5c0cf6ab8e1d8b44fc0dc92d.

**Pull request closed without merging**